src/Symfony/Component/HttpFoundation/Session/Storage/Handler/MongoDbSessionHandler.php

```diff
@@ -12,7 +12,12 @@
 namespace Symfony\Component\HttpFoundation\Session\Storage\Handler;
 
 /**
+ * Session handler using the mongodb/mongodb package and MongoDB driver extension.
+ *
  * @author Markus Bachmann <markus.bachmann@bachi.biz>
+ *
+ * @see https://packagist.org/packages/mongodb/mongodb
+ * @see http://php.net/manual/en/set.mongodb.php
```

Comment:
should use HTTPS

Reply:
this should be changed everwhere then. we use http php.net documentation mostly